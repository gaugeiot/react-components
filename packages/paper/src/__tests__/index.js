import React from "react";
import { render, cleanup } from "@testing-library/react";

import "@testing-library/jest-dom/extend-expect";
import "jest-styled-components";

import Paper from "../index";

afterEach(cleanup);

describe("Paper", () => {
    test("should render", () => {
        const { asFragment } = render(<Paper />);

        expect(asFragment()).toMatchSnapshot();
    });

    test("should render with squared borders", () => {
        const { asFragment } = render(<Paper square />);

        expect(asFragment()).toMatchSnapshot();
    });
});
