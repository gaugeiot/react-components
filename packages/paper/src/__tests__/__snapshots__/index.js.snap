// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paper should render 1`] = `
<DocumentFragment>
  .c0 {
  padding: 0.5rem 0.5rem;
  text-align: start;
  background: #fff;
  border-radius: 4px;
  box-shadow: 0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12);
  color: #000;
  width: 100%;
  height: 100%;
}

<div
    class="c0"
    color="#000"
    height="100"
    width="100"
  />
</DocumentFragment>
`;

exports[`Paper should render with squared borders 1`] = `
<DocumentFragment>
  .c0 {
  padding: 0.5rem 0.5rem;
  text-align: start;
  background: #fff;
  border-radius: 0px;
  box-shadow: 0px 1px 3px 0px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 2px 1px -1px rgba(0,0,0,0.12);
  color: #000;
  width: 100%;
  height: 100%;
}

<div
    class="c0"
    color="#000"
    height="100"
    width="100"
  />
</DocumentFragment>
`;
